tools/persistent_params_stresstesting.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
```

Comment:
So this starts a new thing, with manual test scripts being in `tools/` maybe this should be added to docs in some ways? What goes in tools and why?

Reply:
not sure about that. This is in fact a diagnostics tool so I'm not sure if this needs to be explained in the tools much if others are not using it. We shouldn't document everything, as we don't for the python libraries

Reply:
Perhaps better explanation in the script itself perhaps?

Reply:
I think improving the explanation in the script itself is a good middle ground.